core/src/main/scala/sbt/librarymanagement/LibraryManagementSyntax.scala

```diff
@@ -0,0 +1,24 @@
+package sbt.librarymanagement
+
+abstract class LibraryManagementSyntax extends DependencyBuilders {
```

Comment:
We need it https://github.com/sbt/sbt/blob/1.0.x/sbt/src/main/scala/package.scala#L11